code/src/java/pcgen/io/ExportHandler.java

```diff
@@ -853,7 +853,7 @@ else if (tokenCount != 2)
 			{
 				return leftString.equals(rightString.substring(1));
 			}
-			return leftString.toUpperCase().contains(rightString.toUpperCase());
+			return leftString.toUpperCase(Locale.ENGLISH).contains(rightString.toUpperCase(Locale.ENGLISH));
```

Comment:
This is fairly powerful - and in code I'm not familiar with... how much has this been tested?